index.html

```diff
@@ -0,0 +1,234 @@
+            <ul class="techstack-container">
+              <li>html</li>
+              <li>Javascript</li>
+              <li>Css</li>
+            </ul>
+
+           <button class="project-button">See Project</button></div>
+          </div>  
+    
+        </section>
+    </section>
+
+    <!-- <section class="about"> -->
+    <section class="about">
+        <div class="about-me">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button>Get My Resume</button>
+          </div>
+          <hr>
+          <div class="skills-container">
+
+            <div class="skills-title">
+              <svg width="27" height="26" viewBox="0 0 27 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="0.772461" y="12.7279" width="18" height="18" transform="rotate(-45 0.772461 12.7279)" fill="#FF6B00"/>
+              </svg>
+              <h2>Language</h2>
+            </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Css</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="1f" height="18" fill="#FF6B00"/>
+              </svg>    
+              <h2>Frameworks</h2>
+            </div>
+            
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+            
+            <div class="skills-title">
+              <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+              </svg>     
+              <h2>Skills</h2>
+            </div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>         
+ 
+
+    </section>
+    <!-- contact-form section. -->
+    <section class="contact-section">
+            <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+              <div>
+                <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+                <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30" required>
```

Comment:
- [ ] You did well 😃 at adding the styles, however, I see that you didn't stick to the Figma Design at the design of the submit button and the textarea as stated by the previous reviewer, so please try to remove the border from the textarea and the button submit to be like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/222164315-da733c76-d9a6-46fb-aa18-c4209490f020.png)  |    ![image](https://user-images.githubusercontent.com/83360636/222164260-f5aeefcd-15af-4697-8d90-1cb0c6aa6d50.png)  |